python/packages/ai/tests/prompts/test_prompt_manager.py

```diff
@@ -20,7 +20,7 @@
     UserState,
 )
 
-TEST_ASSERTS_FOLDER = "tests/prompts/test_assets"
+TEST_ASSERTS_FOLDER: str = "tests/prompts/test_assets"
```

Comment:
Thanks for help adding this.
A minor question: I'm just curious about the practice in your mind: should we always add type annotation in our code? You could see some variable declarations does not add type annotation in our code, not sure whether we should update them all.

Reply:
that is a really good question that I have also thought about, We should decide together what practice is best for our team so that we are both on the same page. I _personally_ do the following: use explicit type annotations whenever exposing a type to the user ie options/class properties. I usually use inferred in cases like the one above where its internal only code, or if the language server for some reason gets it wrong which rarely but sometimes happens.

Lets discuss this in more detail during our weekly sync and if we decide to move towards using explict types everywhere I can open a PR to update the codebase and add linting to support it.